app/survey_config/business_config.py

```diff
@@ -48,8 +60,20 @@ def __post_init__(self):
     def get_service_links(
         self, sign_out_url: str, *, is_authenticated: bool
     ) -> Optional[list[dict]]:
+        if self.schema and is_authenticated:
+            ru_ref = get_metadata(current_user).get("ru_ref")  # type: ignore
+            survey_id = self.schema.json["survey_id"]
+            self.account_service_help_url = f"{ACCOUNT_SERVICE_BASE_URL}/surveys/surveys-help?survey_ref={survey_id}&ru_ref={ru_ref}"
+        else:
+            self.account_service_help_url = f"{ACCOUNT_SERVICE_BASE_URL}/help"
+
         return (
             [
+                HeaderLink(
+                    lazy_gettext("Help"),
+                    self.account_service_help_url,
+                    id="header-link-help",
+                ).__dict__,
```

Comment:
Worth assigning this and reusing, as it duplicated below.

Reply:
Assigned a `help_link` var so it's reused, thanks.